src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1369,9 +1373,9 @@ private void updateAuthStatusIconAndText(RemoteOperationResult result) {
     }
 
 
-    private void updateStatusIconFailUserName(){
+    private void updateStatusIconFailUserName(int statusText){

```

Comment:
Please, change the name of the method or add a separate method for maintainance mode and leave this one without parameters. In this state, the name of the method doesn't match semantically what it does.